docs/background/object-storage.md

```diff
@@ -0,0 +1,63 @@
+
+In {{brand}}, you also get multi-tenancy for S3 objects, even though this concept is not intrinsic to S3 itself.
+Here, again, a small conceptual detour applies: when you [create AWS-style credentials](../howto/object-storage/s3/credentials.md), then those are linked to a specific OpenStack user, region, and project.
+When you subsequently use those credentials for S3 interactions, any buckets and objects thus created belong to the project that the credentials are linked to.
+
+
+## Object identity in {{brand}}
+
+Regardless of whether you access a specific object via the Swift API or the S3 API, *it is the same object.*
+This also includes that the object's *container* (accessed via the Swift API) is identical to its containing *bucket* (accessed via the S3 API).
+
+Thus, if you
+
+* [enable the OpenStack CLI](../howto/getting-started/enable-openstack-cli.md) with a certain set of user credentials;
+* use those credentials to [create a private container](../howto/object-storage/swift/private-container.md) named `test` and an object named `my‑object`, using the Swift API;
+* use the same OpenStack credentials to [create AWS-style credentials](../howto/object-storage/s3/credentials.md);
+* then use those AWS-style credentials to list your buckets and objects, using the S3 API,
+
+you will find an S3 bucket named `test`, containing an object named `my‑object`.
+
+
+## Permission and feature conflicts
+
+The fact that objects are identical when accessed via Swift and S3 also entails that bucket and object permissions, set via one API, also apply to object access using the other API.
+
+For example, if you [make a container public](../howto/object-storage/swift/public-container.md) via the Swift API, it also becomes a [public bucket](../howto/object-storage/s3/public-bucket.md) that is accessible via an S3 API path.
+You cannot make a container public via the Swift API, whilst retaining mandatory private (authenticated) access via the S3 API.
+
+Object storage in {{brand}} also does *not* allow you to set competing permissions or features on containers/buckets, based on the API used to access them.
+For example, it is not possible to create [a Swift container that enables versioning](../howto/object-storage/swift/versioning.md), while simultaneously disabling [bucket versioning](../howto/object-storage/s3/versioning/) to the corresponding S3 bucket.
+
+Sometimes, this creates unavoidable conflicts if a specific feature is only available in one of the supported APIs.
+For example, if you enable [object lock](../howto/object-storage/s3/object-lock.md) on an S3 bucket, you cannot modify any lock retention policies from the Swift API, because it simply has no corresponding feature.
```

Comment:
```suggestion
For example, suppose you enable [object lock](../howto/object-storage/s3/object-lock.md) on an S3 bucket. In that case, you cannot modify any lock retention policies from the Swift API because it simply has no corresponding feature.
```